blockprotocol/blockprotocol
# [SITE] Blocks Search API

This PR adds a basic search endpoint for Blocks.


How to Test
- Open Postman or any REST client
- Set the url to `https://blockprotocol-n4g31hg5w-hashintel.vercel.app/api/blocks?q=<any_word>`
- Set a `x-api-key` header and give it any value
- Confirm it returns filtered responses based on the query

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201637454224598
